reframe/frontend/runreport.py

```diff
@@ -187,7 +186,7 @@ def junit_xml_report(json_report):
             attrib={
                 'classname': tid['filename'],
                 'name': casename,
-                'time': str(tid['time_total']),
+                'time': str(decimal.Decimal(tid['time_total'])),
```

Comment:
What was the problem here?

Reply:
I got UnicodeDecodeError, let me retry.

Reply:
The error message is:
```
E   AssertionError: Element 'testcase', attribute 'time': '8.130073547363281e-05' is not a valid value of the atomic type 'xs:decimal'.
```

Reply:
Perhaps, you can add a small comment explaining why we have to use `Decimal`.